frontend/controllers/box_search_controller.rb

```diff
@@ -34,11 +43,11 @@ def perform_search
 
     filters = []
 
-    filters.push({'display_string' => params['indicator']}.to_json) unless params['indicator'].blank?
+    search_params['q'] = "indicator_u_stext:*#{solr_escape(params['indicator'])}*" unless params['indicator'].blank?
```

Comment:
I'm a little uneasy about the use of leading wildcards.  It seems that Solr now supports them by default (previous versions threw an error if you tried), but there's a risk to performance still.  If there are lots of containers like:

    ML-5: Box 1
    ML-5: Box 2
    ML-5: Box 3
    ML-6: Box 1
    ML-6: Box 2
    ML-6: Box 3

Then a query like *1* has to do a linear scan across every indicator term in the index.  It seems like this could end up being a performance concern for hundreds of thousands of boxes.

Do you think it's right that they really want exact matches on numbers?  If we indexed ML-6: Box 3 as tokens ["ML" "6" "Box" "3"] then a search for "3" is still going to do the right thing.  And I'm assuming they don't really want a search for "3" to match "ML-6: Box 300".

If that sounds reasonable, we could split up the indicators into multiple tokens and send them to Solr as a multi-valued field.  We did something like that in yale containers too:

https://github.com/hudmol/container_management/blob/master/indexer/indexer.rb#L91

What do you think?